internal/providers/pke/pkeworkflow/set_master_taint_activity.go

```diff
@@ -62,21 +60,14 @@ func (a *SetMasterTaintActivity) Execute(ctx context.Context, input SetMasterTai
 		return err
 	}
 
-	selector := "node-role.kubernetes.io/master"
-
-	effect := v1.TaintEffectNoSchedule
-	if input.Schedulable {
-		effect = v1.TaintEffectPreferNoSchedule
-	}
-
-	nodes, err := client.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: selector})
+	nodes, err := client.CoreV1().Nodes().List(metav1.ListOptions{LabelSelector: masterKey})
 	if err != nil {
 		return emperror.Wrap(err, "failed to list master nodes")
 	}
 
 	for _, node := range nodes.Items {
 		logger := logger.With("node", node.Name)
-		taints := []v1.Taint{}
+		taints := make([]v1.Taint, 0)
```

Comment:
This creates an empty slice unnecessarily, instead:
```go
var taints []v1.Taint
```
where `taints` will be `nil` which is an empty slice. `append` will allocate a sufficiently sized slice on-the-fly anyway.

Reply:
```suggestion
		var taints []v1.Taint
```